renpy/display/dragdrop.py

```diff
@@ -146,17 +146,26 @@ class Drag(renpy.display.displayable.Displayable, renpy.revertable.RevertableObj
         None.
 
     `clicked`
-        A callback this is called, with no arguments, when the Drag is
-        clicked without being moved. A droppable can also be focused
-        and clicked.  If the callback returns a value other than None,
-        that value is returned as the result of the interaction.
+        A callback that is called when the Drag is clicked without being moved.
+        It is called with one argument, the Drag being clicked on. A droppable
+        can also be focused and clicked. If the callback returns a value other
+        than None, that value is returned as the result of the interaction.
 
     `alternate`
         An action that is run when the Drag is right-clicked (on the
         desktop) or long-pressed without moving (on mobile). It may
         be necessary to increase :var:`config.longpress_duration` if
         this triggers to early on mobile platforms.
 
+    `hovered`
+        An action to run when the drag gains focus.
+
+    `unhovered`
+        An action to run when the drag loses focus.
+
+    `tooltip`
```

Comment:
This (and possibly other additions) need to be added to :args: at the top of the docstring.
I suggest also using ``:scpref:`tooltip` `` as a link to the property.

Also, as written it seems like these get documented as parameters but I don't know they actually are.